ddtrace/llmobs/_experiment.py

```diff
@@ -299,9 +299,8 @@ def run(
 
     @property
     def url(self) -> str:
-        # FIXME: need to use the user's site
-        # also will not work for subdomain orgs
-        return f"https://app.datadoghq.com/llm/experiments/{self._id}"
+        # FIXME: will not work for subdomain orgs
+        return f"{_get_base_url()}/llm/experiments/{self._id}"
```

Comment:
I think `base_url()` is maybe an unnecessary helper to extract out, we can probably keep the logic contained here (there's no other use for extracting/returning the url to users in the LLMObs SDK). Let's just import `ddtrace.config` here directly in this file and move generating the url into this property method

Reply:
updated as per offline discuss